.github/workflows/build.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        node: [8.x, 10.x, 12.x, 14.x]
-        electron: [8.2.0, 9, 10, 11]
+        node: [12.x, 14.x, 16.x]
+        electron: [11, 12, 13]
```

Comment:
Can you explain why older versions of node and electron are not supported anymore?

Reply:
Several reasons:

* Older versions of Node required a hack in `assertStderrIncludes`
* Node 10 and older are [no longer supported](https://nodejs.org/en/about/releases/).
* There is no need to support older Electron versions